src/components/follow/Follow.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC, useState } from 'react';
+import { FormattedMessage, useIntl, IntlShape } from 'react-intl';
+import classnames from 'classnames';
+import Button from '../../elements/button';
+import Icon from '../../elements/icon';
+import ShareModal from '../shareModal';
+import Props from './Follow.types';
+
+const Follow: FC<Props> = ({
+  follower,
+  href,
+  onClickCopyToClipboard,
+  disabled = false,
+  onClickFollow,
+  onClickUnFollow,
+  title,
+}: Props): JSX.Element => {
+  const intl: IntlShape = useIntl();
+  const [isShareModalOpen, setIsShareModalOpen] = useState<boolean>(false);
+  const [followButtonText, setFollowButtonText] = useState<string>(
+    intl.formatMessage({ id: 'toolkit.following' })
+  );
+
+  return (
+    <>
+      <div className="follow-component">
+        <div
+          className="follow-component__button"
+          onMouseOver={() => {
+            setFollowButtonText(intl.formatMessage({ id: 'toolkit.unfollow' }));
+          }}
+          onMouseLeave={() => {
+            setFollowButtonText(intl.formatMessage({ id: 'toolkit.following' }));
+          }}
+          onFocus={() => {}}
+        >
+          <Button
+            className={classnames(
+              { 'follow-btn': !follower.follow },
+              { 'following-btn': follower.follow }
+            )}
+            text={follower.follow ? followButtonText : intl.formatMessage({ id: 'toolkit.follow' })}
+            theme={follower.follow ? 'dark' : 'light'}
+            type="button"
+            extraClass=""
```

Comment:
I think you dont need to pass this prop

Reply:
I can´t remove extraClass prop beacause it sets "btn-undefined" class

Reply:
Can you change Button component code so it does not add that "btn-undefined" if extraClass is not passed? Only add a class if prop exists.

Reply:
Fixed